.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: false
+
+language: node_js
+node_js:
+    - '5.0'
+
+notifications:
+    email: false
+
+install:
+    - npm install
+
+script:
+    - grunt test
```

Comment:
Remove the `script` steps and add this to the `npm test` script in `package.json`:
```json
"scripts": {
  "test": "grunt test"
}
```
and run `npm install --save-dev grunt-cli` to add the grunt binary command to the local npm installs, so `npm test` can run it